runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/ProcessBundleDescriptors.java

```diff
@@ -158,6 +163,29 @@ private static ExecutableProcessBundleDescriptor fromExecutableStageInternal(
         timerSpecs);
   }
 
+  /**
+   * Patches the input coder of a stateful Executable transform to ensure that the byte
```

Comment:
```suggestion
   * Patches the input coder of a stateful transform to ensure that the byte
```

Reply:
I forgot to change the comment. We can add it in a follow-up.